core/data/src/main/java/uk/gov/gchq/gaffer/data/element/function/ElementTupleDefinition.java

```diff
@@ -48,10 +48,8 @@ public String getGroup() {
     @Override
     public Object get(final Object key) {
         Object value = super.get(key);
-        if (null == value) {
-            if (DIRECTED.equals(key)) {
-                value = true;
-            }
+        if (null == value && DIRECTED.equals(key)) {
```

Comment:
```suggestion
        if (value == null && DIRECTED.equals(key)) {
```
I know it's annoying changing it all, but as this code is already being reviewed can we change null == and null != etc to var == null and var != null